problema1_test

```diff
@@ -0,0 +1,36 @@
+require 'test/unit'
+require_relative "problema1"
+
+class MyTest < Test::Unit::TestCase
+
+  # Called before every test method runs. Can be used to set up fixture information.
+  def setup
+    # Do nothing
+  end
+
+  # Called after every test method runs. Can be used to tear down fixture information.
+  def teardown
+    # Do nothing
+  end
+
+  def test_simplu
```

Comment:
Even in Romanian this is not a proper [test naming](http://stackoverflow.com/questions/155436/unit-test-naming-best-practices)